source/dem/particle_wall_linear_force.cc

```diff
@@ -379,51 +382,54 @@ ParticleWallLinearForce<dim>::calculate_linear_contact_force_and_torque(
 
   // Calculation of normal and tangential spring and dashpot constants
   // using particle properties
-  double rp_sqrt = sqrt(particle_properties[DEM::PropertiesIndex::dp] * 0.5);
-
-  double normal_spring_constant =
-    1.0667 * rp_sqrt * this->effective_youngs_modulus[particle_type] *
-    pow((0.9375 * particle_properties[DEM::PropertiesIndex::mass] *
-         contact_info.normal_relative_velocity *
-         contact_info.normal_relative_velocity /
-         (rp_sqrt * this->effective_youngs_modulus[particle_type])),
-        0.2);
-  double tangential_spring_constant =
-    -1.0667 * rp_sqrt * this->effective_youngs_modulus[particle_type] *
-      pow((0.9375 * particle_properties[DEM::PropertiesIndex::mass] *
-           contact_info.tangential_relative_velocity *
-           contact_info.tangential_relative_velocity /
-           (rp_sqrt * this->effective_youngs_modulus[particle_type])),
-          0.2) +
-    DBL_MIN;
-  double normal_damping_constant = sqrt(
-    (4 * particle_properties[DEM::PropertiesIndex::mass] *
-     normal_spring_constant) /
-    (1 + pow((M_PI /
-              (log(this->effective_coefficient_of_restitution[particle_type]) +
-               DBL_MIN)),
-             2)));
+
+
+  double normal_spring_constant = // REF : Coupled CFD-DEM
+    1.2024 * pow(sqrt(particle_properties[DEM::PropertiesIndex::mass]) *
+                   this->effective_youngs_modulus[particle_type] *
+                   this->effective_youngs_modulus[particle_type] *
+                   particle_properties[DEM::PropertiesIndex::dp] * 0.5 *
+                   this->effective_poisson_ratio[particle_type],
+                 0.4);
+
+  double tangential_spring_constant = // REF : MFIX–DEM software
+    normal_spring_constant * 0.4;
+
+  double normal_damping_constant =
+    -2 * log(this->effective_coefficient_of_restitution[particle_type]) *
+    sqrt(particle_properties[DEM::PropertiesIndex::mass] *
+         normal_spring_constant) /
+    sqrt((M_PI * M_PI) +
+         (log(this->effective_coefficient_of_restitution[particle_type]) *
+          log(this->effective_coefficient_of_restitution[particle_type])));
```

Comment:
For me the old model and the one you implemented are the same except the - sign.

Reply:
You are right, the damping is the same. I just didn't realize it at first. When I did, I already made the change. 

In this new version, the minus is necessary since the natural logarithm is at the numerator. 

The first version was probably more efficient. I'll go back to the original.

Reply:
Correction;

Since the beta parameter already exists, I'll keep the equation with the - 2 at the front. :)